### ISSUE TITLE
OC5 Edit additional data url

### ISSUE THREAD

**Issue by Timo-Breumelhof** [sachatrauwaen/OpenContent#196]
To allow editing without the DNN persona bar being visible (with an edit-role in the manifest), it would be nice if users could also edit additional data like categories.
If the editUrl would be part of the model under Additional Data > field, this would allow creating an edit button per additional data type:

![image](https://user-images.githubusercontent.com/8999382/198995985-467c55b7-5134-486d-9fc9-38a115503505.png)

**Comment by Timo-Breumelhof** [comment]
ok sorry, was already implemented before..

![Uploading image.png…]()

**Comment by Timo-Breumelhof** [comment]
I can create a link like this:

```
{{#if Context.IsEditable}}
	
	
   <a href="{{AdditionalData.Context.Categories.EditUrl}}" title="Edit Categories" class="btn btn-primary btn-rounded">Edit Categories</a>

{{/if}}
```

But if I login with a user from the assigned editors role, I get this error when clicking the button:
![image](https://user-images.githubusercontent.com/8999382/200582497-eb063d9f-58c6-43f5-8e55-a34b2e3f10fb.png)

**Comment by Timo-Breumelhof** [comment]
Fixed in 05.00.04
